nucypher/cli/commands/worklock.py

```diff
@@ -264,20 +266,84 @@ def refund(general_config, worklock_options, registry_options, force, hw_wallet)
 
 
 @worklock.command()
+@group_general_config
+@group_registry_options
+@group_worklock_options
+@option_force
+@option_hw_wallet
+@click.option('--gas-limit', help="Gas limit per each verification transaction", type=click.IntRange(min=60000))
+# TODO: Consider moving to administrator (nucypher-deploy) #1758
```

Comment:
What's preventing this from being relocated to `nucypher-deploy` /  Is there some motive for inclusion here?

Reply:
There was some discussion about this in the (option 2) PR: https://github.com/nucypher/nucypher/pull/1732#discussion_r387111897, and https://github.com/nucypher/nucypher/pull/1732#discussion_r386985913.

@vzotova filed a follow-up issue: https://github.com/nucypher/nucypher/issues/1758